config/config.default.js

```diff
@@ -17,9 +17,11 @@ module.exports = () => {
    * @property {Object} xssProtection - 是否开启IE8的XSS Filter，默认开启。
    * @property {Object} csp - csp安全配置。
    * @property {Array} domainWhiteList - 安全跳转白名单
+   * @property {Array} protocolWhiteList - 协议白名单
    */
   exports.security = {
     domainWhiteList: [],
+    protocolWhiteList: [],
```

Comment:
if some rule such as `shtml`, `surl` both has a domain whitelist, will they had difference config value？ if answer is yes, prefer latter one